src/psyclone/tests/parse/kernel_test.py

```diff
@@ -690,7 +776,7 @@ def my_init(self, _):
 def test_get_int_array_section_subscript_err(monkeypatch):
     ''' Check that we raise the appropriate error if the parse tree for the
     LHS of the array declaration is broken. '''
-    from fparser.two import Fortran2003
+
```

Comment:
I would suggest a blank line before the closing quotations instead as in other tests.

Reply:
Done